src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -133,6 +134,50 @@ worker_apply_inter_shard_ddl_command(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * worker_modify_identity_columns takes a table oid, runs an ALTER SEQUENCE statement
+ * for each identity column to adjust the minvalue and maxvalue of the sequence owned by
+ * identity column such that the sequence creates globally unique values.
+ */
+Datum
+worker_modify_identity_columns(PG_FUNCTION_ARGS)
+{
+	CheckCitusVersion(ERROR);
+
+	Oid tableRelationId = PG_GETARG_OID(0);
+
```

Comment:
There should be an ownership check somewhere here. You should only be able to call this for tables that you're the owner of.